crates/sui-indexer-alt-schema/src/objects.rs

```diff
@@ -68,6 +71,43 @@ pub struct StoredCoinBalanceBucket {
     pub coin_balance_bucket: Option<i16>,
 }
 
+impl StoredObjInfo {
+    pub fn from_object(object: &Object, cp_sequence_number: i64) -> anyhow::Result<Self> {
```

Comment:
Can this also be used by the `obj_info` pipeline? It would be good to have just one place to look at for this logic.